@uportal/esco-content-menu/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "esco-content-menu",
+  "version": "0.1.0",
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build --target wc --name esco-content-menu './src/mainwc.js'",
+    "buildold": "vue-cli-service build --name esco-content-menu --target wc 'src/components/EscoContentMenu.vue'",
+    "lint": "vue-cli-service lint"
+  },
+  "dependencies": {
+    "@gip-recia/content-grid": "^0.1.1",
+    "@uportal/content-carousel": "^1.2.0",
+    "@uportal/open-id-connect": "^1.2.0",
+    "@vue/web-component-wrapper": "^1.2.0",
+    "node-sass": "^4.9.0",
+    "sass-loader": "^7.0.3",
+    "vue": "^2.5.16",
+    "vue-awesome": "^2.3.7",
```

Comment:
It's worth noting FortAwesome provides an official font awesome vue component https://github.com/FortAwesome/vue-fontawesome

Reply:
Is there a problem to move to material icons ?
A simple way is to add the font and replace icons by a text only !
Really small to load a font (more as many site use it now so it's cached by the browser)

Reply:
Looks like Vuetify has a Material icon wrapper
https://vuetifyjs.com/en/components/icons

Reply:
Yes but I would to avoid to generate a big js, my goal is to avoid to generate too big js for a web-component, especially since we can do things very simply !